spec/lib/task_helpers/exports/custom_buttons_spec.rb

```diff
@@ -70,7 +70,7 @@
       TaskHelpers::Exports::CustomButtons.new.export(:directory => export_dir)
       file_contents = File.read("#{export_dir}/CustomButtons.yaml")
 
-      expect(YAML.safe_load(file_contents, [Symbol])).to contain_exactly(*custom_button_export_test)
+      expect(YAML.safe_load(file_contents, :permitted_classes => [Symbol])).to contain_exactly(*custom_button_export_test)
```

Comment:
if this PR will take a while to merge, does it make sense to split out these named keyword changes? they seem no-brainer and an easy merge.

(same goes for the `s/eq/include/`)